source/gamemap/MiniMapDrawn.cpp

```diff
@@ -233,14 +233,19 @@ void MiniMapDrawn::update(Ogre::Real timeSinceLastFrame, const std::vector<Ogre:
     //     }
     // }
 
-    mPixelBuffer->lock(mPixelBox, Ogre::HardwareBuffer::HBL_NORMAL);
-
-    Ogre::uint8* pDest;
-    pDest = static_cast<Ogre::uint8*>(mPixelBuffer->getCurrentLock().data) - 1;
+    auto output = mPixelBuffer->lock(mPixelBox, Ogre::HardwareBuffer::HBL_NORMAL);
 
+    size_t xx = 0;
+    size_t yy = 0;
     for(const Color& color : mTiles)
     {
-        drawPixelToMemory(pDest, color.RR, color.GG, color.BB);
+        // TODO: This is probably a bit inefficient at the moment.
```

Comment:
https://ogrecave.github.io/ogre/api/1.10/class_ogre_1_1_hardware_pixel_buffer.html#ae12e40bc3a6461d93b7bfae20e58c866

Reply:
Yup that was what I was thinking about when writing that comment.